src/main/java/com/zextras/carbonio/usermanagement/UserManagementClient.java

```diff
@@ -106,6 +109,47 @@ public Try<UserId> validateUserToken(String carbonioUserToken) {
     }
   }
 
+  /**
+   * Allows to retrieve a list of Carbonio users by {@link UUID} if they exist.
+   *
+   * @param cookie    is a {@link String} representing the requester cookie. (It is necessary to
+   *                  have only the ZM_AUTH_TOKEN cookie).
+   * @param usersUUID is a {@link List} of {@link UUID}s of the Carbonio users to retrieve.
+   * @return a {@link Try#success} containing a {@link List} of {@link UserInfo} representing
+   * the related Carbonio users if they exist. <p />It returns a {@link Try#failure} containing
+   * an {@link UserNotFound} throwable the cookie is not valid. <p />It returns a
+   * {@link Try#failure} containing an {@link InternalServerError} throwable if something goes
+   * wrong during the API call.
+   */
+  public Try<List<UserInfo>> getUsers(
+    String cookie,
+    List<UUID> usersUUID
+  ) {
+    CloseableHttpClient httpClient = HttpClients.createMinimal();
+
+    HttpGet request = new HttpGet(userManagementURL + getUsersEndpoint + usersUUID);
```

Comment:
Please check usersUUID, I don't think you can pass it in this way. The correct way to pass a list of ids in a GET http request is: `/user/?ids=1&ids=2&ids=3`